src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -7,6 +7,7 @@
 
 public class RedoCommand extends Command {
     public static final String COMMAND_WORD = "redo";
+    public static final String MESSAGE_USAGE = "redo";
```

Comment:
This can be better described, maybe something like
redo: redos a previously undo-ed command. Example: `redo`